packages/api/test/fixtures/init-data.sql

```diff
@@ -14,11 +14,11 @@ VALUES ('test-pinning-user', 'test-pinning@user.com', 'test-pinning', 'test-pinn
 INSERT INTO public.user ( name, email, issuer, public_address)
 VALUES ('test-pinning-2-user', 'test-pinning2@user.com', 'test-pinning-2', 'test-pinning-2');
 
-INSERT INTO public.user (id, name, email, issuer, public_address)
-VALUES (6, 'test-pinning-and-restriction-user', 'test-pinning-and-restriction@user.com', 'test-pinning-and-restriction', 'test-pinning-and-restriction');
```

Comment:
Without these change, `db.upsertUser` would somtimes fail with an error that an upsert is trying to insert id=6 where and that violates the uniqueness constraint.